src/components/LayerCollection/LayerPOLine.js

```diff
@@ -35,6 +36,7 @@ class LayerPOLine extends Component {
     delete newLine.created;
     delete newLine.source;
     delete newLine.vendor_detail;
+    delete newLine.eresource;
```

Comment:
it's temporary, just to not break creation of PO Lines within mod-orders-storage API